packages/ui/src/components/accordion/__tests__/accordion.test.tsx

```diff
@@ -1,22 +1,105 @@
-import { render, screen } from '@testing-library/react'
+/* eslint-disable @typescript-eslint/no-explicit-any */
 
-import '@testing-library/jest-dom'
+import { ComponentProps } from 'react'
+
+import { render, RenderResult, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
 import { Accordion } from '../accordion'
 
-describe('Accordion Component', () => {
-  test('renders the accordion with trigger and content', () => {
-    render(
-      <Accordion.Root type="single">
-        <Accordion.Item value="item-1">
-          <Accordion.Trigger>Trigger</Accordion.Trigger>
-          <Accordion.Content>Content</Accordion.Content>
-        </Accordion.Item>
-      </Accordion.Root>
-    )
+const renderComponent = (props: Partial<ComponentProps<typeof Accordion.Root>> = {}): RenderResult =>
+  render(
+    <Accordion.Root type="single" {...(props as any)}>
```

Comment:
Had to do an `as any` since this component has branching types for `single` and `multiple` types. At this point we're already protected by the types when calling `renderComponent`, so we're safe to switch off type checks for this statement.

Reply:
can we disable the lint rule for one line, instead of disabling for the whole file?

Reply:
I was actually going to mirror some other projects we have an disable it for all `.test.ts(x)` files. Those rules are crucial for actual code, but don't make much sense for test files.